src/util.h

```diff
@@ -91,7 +91,7 @@ inline T MultiplyWithOverflowCheck(T a, T b);
 
 namespace per_process {
 // Tells whether the per-process V8::Initialize() is called and
-// if it is safe to call v8::Isolate::GetCurrent().
+// if it is safe to call v8::Isolate::TryGetCurrent().
 extern bool v8_initialized;
```

Comment:
Do we still need this?

Reply:
Yeah … I don’t know why, but `v8::Isolate::TryGetCurrent()` is not something you can call without V8 being initialized first.